cmd/prepare.go

```diff
@@ -0,0 +1,106 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+	"go.temporal.io/server/common/log"
+)
+
+func prepareCmd() *cli.Command {
+	var config PrepareConfig
+	return &cli.Command{
+		Name:  "prepare",
+		Usage: "prepare an SDK for execution",
```

Comment:
(to be used with --prepare-dir) or something meaningful that explains what it means to prepare please

Reply:
I put that info in the README. I think this should remain very brief.